projects/lcms/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eu
-# Copyright 2016 Google Inc.
+# Copyright 2023 Google Inc.
```

Comment:
No need to update copyright in existing files, please leave the year as is (also in the other files).